src/main/kotlin/de/livepoll/api/config/CorsConfig.kt

```diff
@@ -0,0 +1,23 @@
+package de.livepoll.api.config
+
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.config.annotation.CorsRegistry
+import org.springframework.web.servlet.config.annotation.EnableWebMvc
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+
+@Configuration
+@EnableWebMvc
+class CorsConfig: WebMvcConfigurer {
+
+    private val allowedOrigins = setOf(
+            "http://192.168.0.52:4200",
```

Comment:
@marcauberer Do you think it's better to outsource this to a env variable?

Reply:
You're right. I've changed it